src/integrations/prefect-bitbucket/pyproject.toml

```diff
@@ -25,7 +23,8 @@ classifiers = [
   "Topic :: Software Development :: Libraries",
 ]
 dependencies = [
-  "prefect>=2.14.10",
+  "prefect>=2.14.10",                                                 # TODO: update to 3.x
```

Comment:
not sure if theres a better value here in the interim

Reply:
Maybe a path to the package in the repo so that we can install on editable mode by default? We'll need to update it for the RC anyways.